cmd/akashd/genaccounts.go

```diff
@@ -102,7 +103,7 @@ contain valid denominations. Accounts may optionally be supplied with vesting pa
 					genAccount = authvesting.NewDelayedVestingAccountRaw(baseVestingAccount)
 
 				default:
-					return errors.Wrap(ErrInvalidVestingParameters, "must supply start and end time or end time")
+					return errors.New("invalid vesting parameters; must supply start and end time or end time")
```

Comment:
There is a linter that doesn't allow this - you have to pre-declare the error and use `wrap`.